packages/core/src/external/opensearch/fhir-ingestor.ts

```diff
@@ -20,6 +20,9 @@ import { getEntryId } from "./shared/id";
 
 dayjs.extend(duration);
 
+const ERROR_CODES_TO_RETRY = [409, 503];
+const MAX_RETRY_ATTEMPTS = 3;
```

Comment:
Choose one, retry or attempt, please.

E.g., what does this mean, that it will try 3 times or that it will retry 3 times (4 attempts)?

The prop name on that function was chosen accordingly.